mathesar/tests/views/api/test_table_api.py

```diff
@@ -715,6 +715,10 @@ def test_table_404(client):
     assert response.json()['detail'] == 'Not found.'
 
 
+def test_table_not_exist():
```

Comment:
Please remove this test, you're just testing Django's default functionality here, not any of our code.

Reply:
Removed